keycloak/realm.go

```diff
@@ -131,6 +131,9 @@ type Realm struct {
 	WebAuthnPolicyPasswordlessRpId                            string   `json:"webAuthnPolicyPasswordlessRpId"`
 	WebAuthnPolicyPasswordlessSignatureAlgorithms             []string `json:"webAuthnPolicyPasswordlessSignatureAlgorithms"`
 	WebAuthnPolicyPasswordlessUserVerificationRequirement     string   `json:"webAuthnPolicyPasswordlessUserVerificationRequirement"`
+
+	// Roles
+	DefaultRoles []string `json:"defaultRoles"`
```

Comment:
This probably isn't the API we'd want to use to accomplish this.  It looks like `defaultRoles` is only honored on realm creation / update, but it isn't returned when reading.  This is going to result in a perpetual diff for this attribute.

It looks like default realm roles are handled as a composite role, so adding a new default realm role is just modifying the realm's default role to add a new composite role.

When you `GET` a realm, there's a property in the JSON that looks like this:

```json
"defaultRole": {
  "id": "bb6b80f2-0f34-4135-93ca-b3b47f8b7fd4",
  "name": "default-roles-master",
  "description": "${role_default-roles}",
  "composite": true,
  "clientRole": false,
  "containerId": "test"
}
```

So we're going to want to add / remove composites to this role in order to correctly influence the realm's default roles.  You can refer to the `keycloak_role` resource, which also uses composites, for some guidance on how to do this.

Reply:
Yes, my bad. I made it for version 12 and tested it on it... and it worked). But since version 13 api has changed for default_roles.

Now i made changes for 13 and above.